core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -94,5 +98,18 @@ var ExplorationEditorTranslationTab = function() {
     _selectLanguage(language);
     waitFor.pageToFullyLoad();
   };
+
+  this.navigateToTranslationTabFeedback = function() {
+    waitFor.elementToBeClickable(
+      TranslationTabFeedbackButton,
+      'Feedback tab of translation page is not clickable');
+    TranslationTabFeedbackButton.click();
+    waitFor.pageToFullyLoad();
+  };
+
+  this.expectActiveTabToRemainSame = function() {
```

Comment:
make this expectFeedbackTabToBeActive (as `expectActiveTabToRemainSame` sounds more like a general function for all tabs but internally it's checking for the feedback tab.)

Reply:
done